samples/bluetooth/unicast_audio_server/src/main.c

```diff
@@ -205,15 +252,54 @@ static void stream_disconnected(struct bt_audio_stream *stream, uint8_t reason)
 	printk("Audio Stream %p disconnected (reason 0x%02x)\n", stream, reason);
 }
 
+
+#if defined(CONFIG_LIBLC3CODEC)
+
+static void stream_recv_lc3_codec(struct bt_audio_stream *stream, struct net_buf *buf)
+{
+	uint8_t bit_error_detect = 0;
```

Comment:
Given that this is supplied to `Lc3Decoder_run`, I guess we don't need to initialize it?

Reply:
I have not checked that the value is set regardless if there is an error or not. The API do not state that this is set for error-free decoding, hence I prefer to initialize it. (it states it is set !=0 in case of error)